src/js/internal/sql/errors.ts

```
@@ -98,18 +104,23 @@
   sqlState: string | undefined;
 }
 
+interface MySQLError {
+  errno?: number | undefined;
+  sqlState?: string | undefined;
+}
+
 class MySQLError extends SQLError implements Bun.SQL.MySQLError {
```

Biome (2.1.2)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)